addons/context_actions/CfgContext.hpp

```diff
@@ -199,6 +199,29 @@ class EGVAR(context_menu,actions) {
         statement = QUOTE(_hoveredEntity call EFUNC(garage,openGarage));
         priority = -82;
     };
+    class HealUnits {
+        displayName = ECSTRING(modules,ModuleHeal);
+        icon = QPATHTOF(ui\medical_cross_ca.paa);
+        condition = QUOTE(_selectedObjects findIf {crew _x findIf {alive _x} != -1} != -1);
+        class All {
+            displayName = ECSTRING(common,All);
+            statement = QUOTE([ARR_2(_selectedObjects, HEAL_MODE_ALL)] call FUNC(healUnits));
+            priority = 3;
+        };
+        class AI {
+            displayName = ECSTRING(modules,AI);
+            condition = QUOTE(_selectedObjects findIf {crew _x findIf {!isPlayer _x && {alive _x}} != -1} != -1);
+            statement = QUOTE([ARR_2(_selectedObjects, HEAL_MODE_AI)] call FUNC(healUnits));
+            priority = 2;
+        };
+        class Players {
+            displayName = ECSTRING(modules,Players);
+            condition = QUOTE(_selectedObjects findIf {crew _x findIf {isPlayer _x && {alive _x}} != -1} != -1);
+            statement = QUOTE([ARR_2(_selectedObjects, HEAL_MODE_PLAYERS)] call FUNC(healUnits));
+            priority = 1;
+        };
+        priority = -98;
```

Comment:
I think this action might be better suited higher up in the menu, under stance maybe: `priority = -70`?